pkg/controller/namespace/namespace_controller.go

```diff
@@ -468,13 +477,13 @@ func deletePersistentVolumeClaims(kubeClient clientset.Interface, ns string) err
 	return nil
 }
 
-func deleteHorizontalPodAutoscalers(expClient extensions_unversioned.ExtensionsInterface, ns string) error {
```

Comment:
Controller should continue to use extensions client.
We will switch it to autoscaling client in 1.3